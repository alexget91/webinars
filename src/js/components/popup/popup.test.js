import React from "react";
import renderer from "react-test-renderer";
import Popup from "./popup";

it(`Popup correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<Popup title={`title`} onCloseClick={jest.fn()}>
      <div>test</div>
    </Popup>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
