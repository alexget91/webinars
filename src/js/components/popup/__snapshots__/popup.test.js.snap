// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popup correctly renders after relaunch 1`] = `
<div
  className="popup"
>
  <div
    className="popup__overlay"
    onClick={[MockFunction]}
  />
  <div
    className="popup__body"
  >
    <div
      className="popup__close"
      onClick={[MockFunction]}
    />
    <h2
      className="popup__title"
    >
      title
    </h2>
    <div>
      test
    </div>
  </div>
</div>
`;
